modules/core/shared/src/main/scala/lucuma/core/math/units.scala

```diff
@@ -43,8 +43,16 @@ trait units {
   type PicometersPerPixel = Picometer / Pixels
 
   type MetersPerSecond      = Meter / Second
+  given DerivedUnit[MetersPerSecond, Meter / Second, "meters per second", "m/s"] = DerivedUnit()
+  given TypeString[MetersPerSecond] = TypeString("M_S")
```

Comment:
What is `TypeString`

Reply:
> What is `TypeString`

It's a very simple typeclass that specifies a serialization string for units.